### PR TITLE
Update indicator-manage.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line stating that the indicator will get deleted if the expiration time passes.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7245

**Comment by msbemba** [comment]
Two issues in one PR as it is the same document. issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7246

**Review by JohanFreelancer9 — COMMENTED** [review]
Hi @msbemba, could you please clarify or explain what is meant with the second part of the sentence (and whatever happens at the expiration time occur at the second). I do not understand what is meant. Thanks so much.

**Comment by msbemba** [comment]
> Hi @msbemba, could you please clarify or explain what is meant with the second part of the sentence (and whatever happens at the expiration time occur at the second). I do not understand what is meant. Thanks so much.

The expiration format is YYYY-MM-DDTHH:MM:SS.0Z, the command takes effect per the value given in Seconds (SS) and not OZ

File: microsoft-365/security/defender-endpoint/indicator-manage.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ indicatorValue|String|Identity of the [Indicator](ti-indicator.md) entity. **Req
 action|Enum|The action that will be taken if the indicator will be discovered in the organization. Possible values are: "Alert", "AlertAndBlock", and "Allowed". **Required**
 title|String|Indicator alert title. **Required**
 description|String| Description of the indicator. **Required**
-expirationTime|DateTimeOffset|The expiration time of the indicator in the following format YYYY-MM-DDTHH:MM:SS.0Z. **Optional**
+expirationTime|DateTimeOffset|The expiration time of the indicator in the following format YYYY-MM-DDTHH:MM:SS.0Z. The indicator gets deleted if the expiration time passes and whatever happens at the expiration time occur at the second. **Optional**
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/indicator-manage.md]:
```suggestion
expirationTime|DateTimeOffset|The expiration time of the indicator in the following format YYYY-MM-DDTHH:MM:SS.0Z. The indicator gets deleted if the expiration time passes and whatever happens at the expiration time occurs at the seconds (SS) value. **Optional**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Thank you, Sri. Kindly commit the changes if you agree with them.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**